secrets-provider/aws/index.ts

```diff
@@ -10,12 +10,36 @@ const config = new pulumi.Config();
 const bucketName = config.require('bucketName');
 const secretValue = config.requireSecret('secretValue');
 
+export function configureACL(bucketName: string, bucket: aws.s3.BucketV2, acl: string): aws.s3.BucketAclV2 {
+    const ownership = new aws.s3.BucketOwnershipControls(bucketName, {
+        bucket: bucket.bucket,
+        rule: {
+            objectOwnership: "BucketOwnerPreferred",
```

Comment:
`BucketOwnerEnforced` (All ACLs disabled), the default setting for S3 buckets, is preferred if no public access is needed.